GentApp/Services/CompanyService.cs

```diff
@@ -27,20 +27,35 @@ class CompanyService {
 			return Enumerable.Empty<Company>();
 		}
 
-		public async Task Save(Company company) {
+		public async Task<Company> Save(Company company) {
 			try {
-				await HttpClient.PostAsync(apiUrl, new StringContent(JsonConvert.SerializeObject(company)));
```

Comment:
wrm dit veranderd, is toch beter zo?

Reply:
keb alles hetzelfde gedaan. Zo met eerst je request samenstellen en dan sendasync te gebruiken.